src/app/legacy/containers/Header/index.jsx

```diff
@@ -76,7 +76,15 @@ const HeaderContainer = ({
     </SkipLink>
   );
 
-  const shouldRenderScriptSwitch = scriptLink && renderScriptSwitch;
+  let shouldRenderScriptSwitch;
```

Comment:
Could maybe initialise this to `false`. `undefined` should have the same behaviour, but perhaps just for completeness its best to have it as a consistent type.